stemcell_builder/stages/blobstore_clis/config.sh

```diff
@@ -11,7 +11,7 @@ persist_value downloaded_cli_dir
 rm -rf "${downloaded_cli_dir}"
 mkdir -p "${downloaded_cli_dir}"
 
-blobstore_clis=(bosh-blobstore-dav bosh-blobstore-gcs bosh-blobstore-s3)
+blobstore_clis=(bosh-blobstore-dav bosh-blobstore-gcs bosh-blobstore-s3 bosh-blobstore-az)
```

Comment:
Please adjust the package name to `bosh-blobstore-azure-storage` corresponding to the definition in azure-storage-cli repo.

Reply:
Done.